core/domain/feedback_jobs_one_off.py

```diff
@@ -104,6 +104,24 @@ def _cache_last_nonempty_message_author_id(
         return False
 
 
+class TextMessageLengthAuditOneOffJob(jobs.BaseMapReduceOneOffJobManager):
+    """Job that audits and validates text length"""
+
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [feedback_models.GeneralFeedbackMessageModel]
+
+    @staticmethod
+    def map(model_instance):
+        if model_instance.text:
+            if len(model_instance.text) > 10000:
+                yield (model_instance.thread_id, model_instance.message_id)
```

Comment:
Btw, I would recommend adding a `yield ('SUCCESS', 1)` for EVERY models so that we can verify that the job was run on a correct number of models.

Reply:
Done.